src/EnergyPlus/MoistureBalanceEMPDManager.cc

```diff
@@ -599,7 +600,7 @@ namespace MoistureBalanceEMPDManager {
 		}
 		// Calculate resistance between surface-layer/air interface and center of surface layer. [s/m]
 		// This is the physical surface of the material.
-		RSurfaceLayer = 1.0 / hm_surf_layer - 1.0 / h_mass_conv_in_fd - Rcoating;
+		RSurfaceLayer = 1.0 / hm_surf_layer - 1.0 / h_mass_conv_in_fd;
```

Comment:
Can you clarify what this is doing?  The h terms seem to be conductance like, such that resistances are added/subtracted as reciprocals, but why the subtraction here?  Taking a previously calculated total resistance and shaving off some of it?  I'm not saying it's wrong, but it is odd.